ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -601,21 +601,41 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
-  handleLockChange(time) {
+  handleLockChange(autoLockTimeLimitBeforeNormalization) {
     const { t } = this.context;
-    const autoLockTimeLimit = Math.max(Number(time), 0);
 
-    this.setState(() => {
-      let lockTimeError = '';
+    if (autoLockTimeLimitBeforeNormalization === '') {
+      this.setState({
+        autoLockTimeLimitBeforeNormalization,
+        autoLockTimeLimit: '5',
```

Comment:
I guess this is the default value. Was this in effect before as a default?

Reply:
It has already sort of been the default; I noticed that the `placeholder` attribute of the input is set to "5". However, we never effectively used the default because it has been impossible for users to clear the field (it always appear as "0"). I suppose to be consistent with the current behavior we could change the placeholder to "0" instead of doing this.

Reply:
Leaving the default as zero makes sense, especially if that's already how it works in practice today.

Changing the default to 5 might result in this setting remaining on when users try to disable it. e.g. if I clear the field, I probably don't expect that to result in "5" being entered.

Reply:
Edit: [It looks 0 is the default according to `advanced-tab.container.js`.](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/settings/advanced-tab/advanced-tab.container.js#L40) Following this and the current behavior I've changed the placeholder to 0 and the default value to 0 here: [511b2f2](https://github.com/MetaMask/metamask-extension/pull/19653/commits/511b2f25b815c5de1e7eb0a4c4a1628943f4b3cf)